src/platforms/android/common/gradle.mdx

```diff
@@ -34,12 +34,12 @@ buildscript {
 }
 
 plugins {
-    id("io.sentry.android.gradle") version "{{ packages.version('sentry.java.android.gradle-plugin', '2.0.0') }}"
+    id("io.sentry.android.gradle") version "{{ packages.version('sentry.java.android.gradle-plugin', '3.0.0') }}"
 }
 ```
 <Note>
 
-The `io.sentry.android.gradle` >= `3.0.0` requires [Android Gradle Plugin >= 7.0.0](https://developer.android.com/studio/releases/gradle-plugin#7-0-0).
+The `io.sentry.android.gradle` >= `3.0.0` requires [Android Gradle Plugin >= 7.0.0](https://developer.android.com/studio/releases/gradle-plugin#7-0-0). For an older version of the Android Gradle Plugin, consider using `io.sentry.android.gradle:2.+`.
```

Comment:
`l`: Do we know what the older version is? Can we be more specific here?

Reply:
it's basically many, so Isabel's suggestion made it clearer I think